phi/tools/linear_tools.py

```diff
@@ -0,0 +1,254 @@
+import requests
+from os import getenv
+from phi.tools import Toolkit
+from phi.utils.log import logger
+
+
+class LinearTool(Toolkit):
+    def __init__(self):
+        super().__init__(name="linear tools")
+        self.api_token = getenv("LINEAR_API_KEY")
```

Comment:
Add if condition to check if the key exists. If it doesn't exists then raise an error